worker/app/tasks/base.py

```diff
@@ -43,7 +43,7 @@ def set_up(self):
 
     def get_dns(self):
         """list of DNS IPs to feed `dns` on scrappers with: [<dnscache_ip>]"""
-        if not hasattr(self, 'run_dnscache'):
+        if not hasattr(self, 'run_dnscache') or self.task_id not in self.run_dnscache._container_name:
```

Comment:
private property access is bad

Reply:
right, followed convention when created it but it doesn't make sense to keep this one private. changed.